backbone.js

```diff
@@ -1633,9 +1523,8 @@
 
   // Routers map faux-URLs to actions, and fire events when routes are
   // matched. Creating a new one sets its `routes` hash, if not set statically.
-  var Router = Backbone.Router = function(options) {
+  var Router = Backbone.Router = function (options) {
     options || (options = {});
-    this.preinitialize.apply(this, arguments);
```

Comment:
this is removing all of the changes currently in `master`

Reply:
I forked from master. What did I miss????